config/helmet-csp.js

```diff
@@ -15,10 +15,7 @@ const CSP = {
         'platform.twitter.com',
         'cdn.syndication.twimg.com',
         'api.github.com',
-        'cdn.carbonads.com',
-        'srv.carbonads.net',
         'adn.fusionads.net',
```

Comment:
I think we can remove this one too.

Reply:
what if we need to revert?

Reply:
We'll revert the whole patch anyway, so I'd rather have this complete.